signer/xpi/cose.go

```diff
@@ -347,8 +349,8 @@ func (s *XPISigner) issueCOSESignature(cn string, manifest []byte, algs []*cose.
 	if s.issuerCert == nil {
 		return nil, fmt.Errorf("xpi: cannot issue COSE Signature when XPISigner.issuerCert is nil")
 	}
-	if len(s.issuerCert.Raw) < 1100 {
-		return nil, fmt.Errorf("xpi: cannot issue COSE Signature DER encoded XPISigner.issuerCert should be at least 1100 bytes long")
+	if len(s.issuerCert.Raw) < 600 {
```

Comment:
Why this change? and is there a test for this?

Reply:
The value is a heuristic for "probably a real cert." It was an ask in the original COSE PR:

> you already verify that issuerCert != nil above, so this is more of a nit, but maybe check that len(s.issuerCert.Raw) > X where X is a minimal value for a cert?

https://github.com/mozilla-services/autograph/pull/76/files#r200813777



From the commit message history:

https://github.com/mozilla-services/autograph/pull/722/commits/8b0550205afcc2b3a58a38edeb849978044a88fb:

> ECDSA certs can be shorter
>
> The main check is that it's not empty and reasonable length the previous value based on a ballpark guess of dev certs at the time: 6365141

https://github.com/mozilla-services/autograph/commit/636514147dd70291bbec2a86519715bf983a20e7:

> I don't know the minimum offhand, but the dev ones are 2007 for the rsa one and 1132 for the ecdsa.
> 
> refs:
> https://github.com/mozilla-services/autograph/pull/76/files#r202278633
> https://github.com/mozilla-services/autograph/pull/76/files#r200813777

Reply:
Ok sounds good ty!